geomstats/datasets/sample_sdp_2d.py

```diff
@@ -0,0 +1,131 @@
+"""Helper data classes for the MDM illustration example on SPD matrices."""
+
+import geomstats.backend as gs
+from geomstats.geometry.spd_matrices import SPDMatrices
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+
+
+class DatasetSPD_2D():
+    """
+    Sample 2D SPD dataset.
+
+    Data is of shape [n_samples * n_classes, n_features];
+    Labels are of shape [n_samples * n_classes, n_classes].
+    """
+
+    def __init__(self, n_samples=100, n_features=2, n_classes=3):
+        self.n_samples = n_samples
+        self.n_features = n_features
+        self.n_classes = n_classes
+        self.data_helper = DataHelper()
+
+    def generate_sample_dataset(self):
+        """
+        Generate the dataset.
+
+        :return:
+        X: array-like, shape = [n_samples * n_classes, n_features]: data
+        Y: array-like, shape = [n_samples * n_classes, n_classes]: labels
+        """
+        X, Y = self.setup_data()
+        X, Y = self.data_helper.shuffle(X, Y)
+        return X, Y
+
+    def setup_data(self):
+        """
+        Generate the un-shuffled dataset.
+
+        :return:
+        X: array-like, shape = [n_samples * n_classes, n_features]: data
+        Y: array-like, shape = [n_samples * n_classes, n_classes]: labels
+        """
+        mean_covariance_eigenvalues = gs.random.uniform(
+            0.1, 5., (self.n_classes, self.n_features))
+        base_rotations = SpecialOrthogonal(n=self.n_features).random_gaussian(
+            gs.eye(self.n_features), 1, n_samples=self.n_classes)
+        # var_eigenvalues = gs.random.uniform(
+        #     .04, .06, (self.n_classes, self.n_features))
+        var_rotations = gs.random.uniform(
+            .5, .75, (self.n_classes))
+
+        # data
+        cov = gs.zeros(
+            (self.n_classes *
+             self.n_samples,
+             self.n_features,
+             self.n_features))
+        Y = gs.zeros((self.n_classes * self.n_samples, self.n_classes))
```

Comment:
Lower-case variable names for variables, the uppercase letters are reserved for class names. The only exception that we make is the design matrix `X`, in order to follow scikit-learn syntax.

So: `Y` -> `y` (which is also lower case in scikit learn)?